### DISCUSSION TITLE
Matrix Channel

### DISCUSSION THREAD

**Discussion by LongnoseRob** [pymeasure/pymeasure#745] [Polls]
Many opensouce projects use a chatroom like IRC or Matrix for more realtime communciation beside issues and PR.

Would something like this find any interest in this project?

It coiuld be used for (without any specific order):

- stratregic discussions
- quick idea exchange
- experience exchange
- deep technical discussions on implementation details and side-effects, etc
- test-coordination of new features or before releases

**Comment by bilderbuchi** [comment]
I created a public/open Matrix "Space" with a General chat room here: https://matrix.to/#/#pymeasure:matrix.org
Note that we maintainers won't necessarily monitor this chat room for bug reports or somesuch, we'd rather not add yet _another_ source of communication to our watch list.  ;-)

**Reply by LongnoseRob** [reply]
Great!
